modules/admin_manual/pages/maintenance/backup.adoc

```diff
@@ -6,12 +6,12 @@
 
 If you are running ownCloud in a docker container, refer to xref:installation/docker/index.adoc#upgrading-owncloud-on-docker[Upgrading ownCloud on docker].
 
-When backing up your ownCloud server, there are four things you need to copy:
+When backing up your ownCloud server, you need to copy four things in your ownCloud folder (by default located under `/var/www/`):
 
-. the `config/` directory.
-. the `data/` directory,
+. the `config/` directory containing the main configuration file `config.php` plus sample configurations and possibly custom configurations,
+. the `data/` directory containing web-server-related files, `owncloud.log` and possibly encryption keys,
```

Comment:
After re-reading this, I am still not happy with that sentence because the data directory can be huge due to the user files. Maybe we have to ask PM which elements exactly to be looked for being saved.

Reply:
Right, but that might take some time, so let's go with the complete data directory for now.